### ISSUE TITLE
Basics Metrics Addition

### ISSUE THREAD

**Issue by Jimmy062006** [Jimmy062006/Opux#35]
I'd like to add some really really basic metrics to to Opux so that I can see the amount of active Bots there are. I don't however want to add it without first consulting you guys that do actively use it.

The idea is that once a day Opux will access a webpage and send a small non-identifying string this string will be set when you first start the bot and not contain and data or encrypted data it will merely be a random string to identify your client so that it is not counted twice and allows multiple clients.

This is only because I'm interested in the amount of people using Opux.

Please leave you opinions below.

**Comment by shibdib** [comment]
Open sources code with the phone home stuff clearly defined for all to see should be fine imo..

But it's also eve and everyone is opsec crazy (and a phone home with a discord bot could pull absurd amounts of intel if it was configured to do so which makes people nervous)

**Comment by Rayvenhaus** [comment]
I'm ok with this.

**Comment by Jimmy062006** [comment]
@shibdib I'm aware of that and that's why this question is here. It would be very basic and clear to see with just a one line URL call `http://exampleurl.com/?id=ujewhrifuenbr` as an example

**Comment by tim7433** [comment]
this would be fine with me. i would say you should allow for a admin to be able to OPT OUT of this tho.

**Comment by k9by** [comment]
I agree, full disclosure on what you are sending along with the ability for admins to "OPT OUT" seems standard practice in today's space. One fear is, that today you only send a guid, but what about 18 months from now...

**Comment by Jimmy062006** [comment]
@lostboy1 18 months from now I'll be sending at most just a "GUID" lol I've no intention of trying to spy on people and I'm happy to show the two lines of code that I'd add to achieve it.
